pack.ps1

```diff
@@ -0,0 +1,146 @@
+<#
+Create a NuGet package from an output location
```

Comment:
does the pack.ps1 get called in post build?

Reply:
NuGet packing should be done as part of the release process.